Add documentation for responsive app setup (#717)
SAP/luigi: docs/luigi-ux-features.md

However, you can render the Luigi content in any other HTML container. It can be useful if you want to add a header or a footer on top of the Luigi content. To use this feature, add the `luigi-app-root` custom HTML attribute to the HTML tag in which you want to render the Luigi content.

>**NOTE:** If you render the Luigi content in a custom container, the container is positioned relatively when you apply your own CSS. Also, set the height of the Luigi custom container either in **px** or **vh**.

> ### Responsive app setup

Reviewer: ```suggestion
### Responsive application setup
```

> You can quickly adjust the Luigi application to improve user experience on mobile devices like smartphones or tablets.

Reviewer: ```suggestion
You can quickly adjust the Luigi application to improve user experience on mobile devices, such as smartphones or tablets. Here are some examples:
```

> There are a couple quick adjustments that can be applied to the Luigi application, that will improve the user experience on mobile devices like smartphones or tables.

Reviewer: ```suggestion
You can quickly adjust the Luigi application to improve user experience on mobile devices like smartphones or tablets.
```

> In first place, it is recommended to add following to your _index.html_, for the Luigi app to be best rendered on a mobile device.

Reviewer: ```suggestion
* Add the following line to your `index.html` file for the Luigi application to render well on a mobile device:
```

```html
<meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1">
```

> * Make the left navigation responsive by applying [additional settings](./general-settings.md)

Reviewer: ```suggestion
* Define and apply [additional settings](./general-settings.md) to make the left navigation responsive.
```

> Apart from that, it is also possible to make the left navigation responsive. To do so, some additional settings need to be applied as specified [here](./general-settings.md)

Reviewer: ```suggestion
* Make the left navigation responsive by applying [additional settings](./general-settings.md)
```
